python/lsst/sims/GalSimInterface/galSimCelestialObject.py

```diff
@@ -40,6 +40,10 @@ def __init__(self, galSimType, sed, ra, dec, xPupil, yPupil,
         @param [in] positionAngle is the position angle of the object in radians
 
         @param [in] sindex is the sersic index of the object
+
+        @param [in] fluxDict is a dict of photon count (not ADU) values keyed to bandpass names,
```

Comment:
I think you are correct.  I am having a hard time wrapping my head around the relationships between ADU, photons, and electrons.  I think we assume here that quantum efficiency is wrapped up in the bandpass throughput grid.